content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -189,14 +188,12 @@ Please note that:
 
 ### Generate static Pod manifests for control plane components
 
-Kubeadm writes static Pod manifest files for control plane components to `/etc/kubernetes/manifests`; the kubelet watches this directory for Pods to create on startup.
+Kubeadm writes static Pod manifest files for control plane components to `/etc/kubernetes/manifests`. The kubelet watches this directory for Pods to create on startup.
 
 Static Pod manifest share a set of common properties:
 
 - All static Pods are deployed on `kube-system` namespace
 - All static Pods gets `tier:control-plane` and `component:{component-name}` labels
-- All static Pods gets `scheduler.alpha.kubernetes.io/critical-pod` annotation (this will be moved over to the proper solution
```

Comment:
> - All static Pods use the `system-node-critical` priority class

(note this was changed in 1.19)

Reply:
Looks like the annotation was removed in v1.16 - https://github.com/kubernetes/kubernetes/commit/495faa22db7e58c2d92704d48f77acebc7a831b0

Reply:
And the priority class was updated to `system-node-critical` in https://github.com/kubernetes/kubernetes/commit/83a59c02f9b6fa29a352043509fca076c05bdb0e by you :)

Reply:
i think having:
> - All static Pods use the system-node-critical priority class

is fine here because it's accurate for the master branch.

Reply:
SG. marking as resolved.